tests/unit_tests_old/tests_launch/test_launch_kubernetes.py

```diff
@@ -371,7 +372,8 @@ def test_launch_kube_failed(
     )
     project = "test"
     entity = "mock_server_entity"
-    pod_name = f"launch-{entity}-{project}-asdfasdf"
+    entity_clean = "mockserverentity"
+    pod_name = f"launch-{entity_clean}-{project}-asdfasdf"
```

Comment:
if we are being really picky we could make a simple utility function to like:
```python
def make_test_pod_name(entity, project):
    return f"launch-{entity.replace('_', '')}-{project}-asdfasdf"
```

Reply:
idk it might look weird otherwise, or maybe just put a comment

Reply:
I think if we make a utility function `make_name_dns_safe` we could use that utility function in both code and tests?